### ISSUE TITLE
AMD Radeon RX 550 / 550 Series - works as advertised on Windows10, but not Linux

### ISSUE THREAD

**Issue by hwpfeil** [FoldingAtHome/fah-issues#1699]
## Your Environment
* **F@H Software version**: 7.6.21
* **Operating System**: Dual-boot Windows10 and Linux
Graphics Processor: AMD Radeon RX 550 / 550 Series
KDE Plasma Version: 5.26.4
Kernel Version: 5.19.17 (64-bit)
## Expected Behavior
    GPU folding should work on both Slackware-current and Windows10
## Current Behavior
    GPU folding runs on Windows10, but not on Linux

**Comment by joe-h-fah** [comment]
Just a note that Slackware is not a supported distribution for the Folding@home client software.  Supported distributions are Red Hat, Ubuntu, OpenSUSE, and derivatives of those.

GPU folding depends on a full installation of the proprietary drivers and the matching OpenCL support.  The open source drivers and Mesa package to support OpenCL are not sufficient.  This is not an issue with the client itself.  In addition, the AMD drivers may have kernel dependencies

**Comment by hwpfeil** [comment]
I get around your shortcomings by grabbing the Fedora packages, extracting them all into one folder, then I run the `/sbin/makepkg` script. Been doing it for years.

The question should arise, why am I now reporting this? I've been folding for years. For reasons as yet unknown to me, all previous versions ran GPU folding on Linux. Then after a recent "upgrade", GPU folding stopped working. Let me rephrase that. GPU folding on Linux worked; now GPU folding on Linux does not work. This is indeed an issue with the client itself. Same Mesa; no AMD drivers, different FAH version.
